### DISCUSSION TITLE
I´m the only one, or have somebody also have this problem?

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#2692] [Help]
Hello,
meanwhile I think I´m the only one with this issue. I have a very solid System, Ryzen 7 3700X, RTX2070Super, 32GB (3600er) 1TB SSD, only FS on the system, all up-to-date, and good performance all over the Sim in nearly every situation. BUT:
When i fly with the A32Nx mod (with or without the FBW-mod) i recognise massive periodic stutters and drops in CoherendGTDraw values. BUT only when waypoints are in the FMC. moreover, the more waypoints, the more intense the Problem is, as you can see in the Video. It leads to periodic stutter in the Cockpit and massively degrades the flying experiance. 
https://www.youtube.com/watch?v=Gz6MBc2UNP8 (look @ the CoherentGTDraw value) 

I know, it is a "complex" aircraft and its normal to have some less FPS than in a default cessna, but, as I said before, FPS are good and no stutters, when the FMC - Flightplan is empty.

Does anybody can confirm this, or is it just on my maschine?

**Comment by Daboj1337** [comment]
Yes ive got exact the same issue! After i insert a flight plan in the mcdu ive got massiv stutters with my i9 10900k and 3090ti!
